docs/docs/adding-analytics.md

```diff
@@ -37,16 +37,20 @@ npm install --save gatsby-plugin-google-analytics
 ```
 
 ```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: 'gatsby-plugin-google-analytics',
```

Comment:
This is fine, but as an FYI template literals are synonymous with strings.

Reply:
fixed